src/dbnode/storage/series/buffer.go

```diff
@@ -52,8 +52,8 @@ var (
 )
 
 const (
-	bucketsCacheSize = 2
-
+	bucketsCacheSize  = 2
+	timesArraySize    = 8
```

Comment:
more a question: how'd you arrive at 8?

Reply:
Mostly just a guess. This represents the maximum number of blocks we think we'll be flushing at one time. We don't really know what out of order writes will be like for now. I'll put in a todo to figure out if 8 is sane.

Reply:
Sounds like a plan. If you find a lot of variance, we’ll just have to pool the evictedTimes later